File: Frontend/src/services/Dashboard/dashboardService.js
// src/services/Dashboard/dashboardService.js
import { api } from "../http";
import { DASHBOARD_PREFIX } from "../../constants/api";

export const dashboardService = {
  getTodaySales: () => api.get(`${DASHBOARD_PREFIX}/sales/today`),
  getTodayTickets: () => api.get(`${DASHBOARD_PREFIX}/tickets/today`),
  getTodayVisits: () => api.get(`${DASHBOARD_PREFIX}/visits/today`),
};

File: Backend/DigiTicket/src/main/java/com/digiticket/service/dashboard/DashboardService.java
package com.digiticket.service.dashboard;

import com.digiticket.dto.dashboard.DailySalesDTO;
import com.digiticket.dto.dashboard.DailyTicketsDTO;
import com.digiticket.dto.dashboard.DailyVisitsDTO;

public interface DashboardService {
    DailySalesDTO getTodaySales();
    DailyTicketsDTO getTodayTickets();
    DailyVisitsDTO getTodayVisits();

}
